Engine: BSD: Address SIGSEGV error on Loading / Creating a Project

[valgrind-log.zip](https://github.com/user-attachments/files/16077941/valgrind-log.zip)
## Description

The editor partially loads the project and then crashes after a few seconds. It immediately crashes after project creation.

## Prerequisites

Was on the project selection screen waiting to open a project.

## Steps to Reproduce

1. Build latest source code with CMake.
2. Install to the SDK build directory using `gmake install`
3. Run WorldEditor from inside of `install-root/sdk/2024.2/freebsd/x86/bin/` directory
4. Load up or create new project.

### Expected behavior:

Full editor loads.

### Actual behavior:

Crashes while loading editor and engine.

### Reproducibility:

Consistently.

## Versions

![Version](https://github.com/thunder-engine/thunder/assets/59133880/2d2de265-dfa2-4f56-9f21-34230a017b46)

## Additional Information

I decided to run both gdb and Valgrind to get to the bottom of this. Attached is the Valgrind log. 

Note the `Invalid write of size 8` on line 400 and then the `Address 0x70 is not stack'd` on line 413. There's other errors but `MaterialInstance::setTransform` is the crashing function in both debugger applications.

Running `gmake install` produces this:
```
CMake Error at build/cmake_install.cmake:45 (file):
  file INSTALL cannot find "/plugins/imageformats": No such file or
  directory.
Call Stack (most recent call first):
  cmake_install.cmake:48 (include)
```
and then it complains about not having a local Qt installation and errors out with that too.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -52,7 +52,7 @@ string(TIMESTAMP YEAR "%Y" UTC)
 
 # Set project properties
 set(COPYRIGHT_YEAR ${YEAR})
-set(RESOURCE_ROOT "../worldeditor/bin/")
+set(RESOURCE_ROOT "${CMAKE_SOURCE_DIR}/worldeditor/bin/")
 
 set(PREFIX "")
 set(LAUNCHER_PATH "launcher")

